Sources/HAP/Server/Device.swift

```diff
@@ -30,11 +30,15 @@ struct Box<T: Any>: Hashable, Equatable {
     }
 }
 
+
 public class Device {
     public let name: String
-    public let setupCode: String
     public let isBridge: Bool
-
+    
```

Comment:
You can configure Xcode to automatically trim trailing whitespace, that would result in less whitespace changes.

Reply:
OK, I'll look for that